### ISSUE TITLE
Update to 1.5.0

### ISSUE THREAD

**Issue by MechEng70** [shbatm/hacs-isy994#66]
Attempted to upgrade to 1.5.0.  After the install, the logs mentioned that the isy994 section in the configuration.yaml was not needed.  Makes sense, changelog indicates that as of 1.4.0 removed the need for the configuration.yaml.
However, none of the devices were active.  Even rebooted and none were active.  Do I need to remove the isy section prior to installing?  Would this affect the install or just indicate it is not needed?

Removed the isy994: isy_variables from the configuration and installed 1.5.0.  Here is the error message.

```
The following integrations and platforms could not be set up:

isy994.binary_sensor
isy994.cover
isy994.lock
isy994.switch
isy994.climate
isy994.light
Please check your config.
```

**Comment by shbatm** [comment]
Apologies--some of the changes made will only work with Home Assistant 0.110.0.  I've updated the minimum version for HACS.

**Comment by MechEng70** [comment]
I do have 109.3. I will revert back to 1.3.7 until 110 is rolled out.

**Comment by MechEng70** [comment]
I am trying to reinstall the 1.3.7 and it comes up with the 110 requirement.

Will not allow me to revert back.

**Comment by shbatm** [comment]
Try an uninstall from HACS and reinstall the lower version.

Also, 110 beta is available, if you're feeling brave...
https://rc.home-assistant.io/blog/2020/05/12/release-110/

**Comment by MechEng70** [comment]
I had to reload the page a couple of times and then is allowed me to install 1.3.7.  

I'll wait until 110 is released for maybe even 110.1. 😁

Thanks!

**Comment by gormic75** [comment]
Did something change for the programs? I have few covers defined in HA.cover and HA.switch folders on the isy but they are not recognized anymore after upgrade to 110.0.

**Comment by shbatm** [comment]
@gormic75 Not intentionally. Mine were still loading.

Do you have any warnings in your log? Especially any like `Program cover entity 'garage' not loaded...`

**Comment by shbatm** [comment]
I just confirmed, my switches and covers are still appearing in Home Assistant.

Are you not seeing them at all, or are you seeing what you had on your Frontend show up as `unavailable`? If the latter, check that the entity name didn't change for some reason (look in Developer Tools > States and see if they're there under a different name.

**Comment by gormic75** [comment]
I don't see anything in the log related to isy. All of the entities are listed as unavailable in the Frontend and in the States:

cover.garage_door_cover | unavailable | restored: true supported_features: 15 friendly_name: Garage Door Cover
-- | -- | --

One thing, when I upgraded to 110 I had the hacs 1.3.7 installed and removed that after the first reboot. I uninstalled it and rebooted, I got few new entities like a battery level that was missing for my motion sensor but the cover stopped being available.
